modulefiles/GDAS/wcoss2.intel.lua

```diff
@@ -33,13 +33,17 @@ load("nco/5.0.6")
 load("gsl/2.7")
 load("prod_util/2.0.14")
 load("bufr/12.0.1")
+load("fms-C/2023.04")
 
 -- hack for pybind11
 setenv("pybind11_ROOT", "/apps/spack/python/3.8.6/intel/19.1.3.304/pjn2nzkjvqgmjw4hmyz43v5x4jbxjzpk/lib/python3.8/site-packages/pybind11/share/cmake/pybind11")
 
 -- hack for git-lfs
 prepend_path("PATH", "/apps/spack/git-lfs/2.11.0/gcc/11.2.0/m6b6nl5kfqngfteqbggydc7kflxere3s/bin")
 
+-- hack for FMS
```

Comment:
Why do we need a FMS hack?  Are we waiting for something to be installed on WCOSS2?

Reply:
Yes.  spack-stack should help, but we should ask WCOSS2 sysadmins as well.
@CoryMartin-NOAA, Is `fms_ROOT` defined?  If so, that should be sufficient.

Reply:
I'll have to double check, but it looked like nothing was defined in the modulefile besides appending paths